### PR TITLE
DCS-714 Moving to node 14

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

**Review by JamieDraperUK — DISMISSED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -15,16 +15,16 @@ RUN addgroup --gid 2000 --system appgroup && \
 
 WORKDIR /app
 
-RUN apt-get update && \
-    apt-get upgrade -y && \
```
Comment (petergphillips) [inline Dockerfile:19]: don't you still need this in the `base` to ensure that the final image contains up to date libraries?

Comment (andrewrlee) [reply Dockerfile:19]: yes! fixed this

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**